app/views/employees/edit.html.erb

```diff
@@ -2,6 +2,12 @@
 <h3>Update Employee</h3>
 </div>
 
+<p id="info"><%= info %></p>
```

Comment:
If the `info` message is being rendered here, it is not duplicated in the code block below?

Reply:
I will delete this info, we decided to use the second one because it covers more validations.